### PR TITLE
https://issues.redhat.com/browse/ACM-4792 Update on-prem

### PR DESCRIPTION
**Pull request by oafischer**
2.7 and 2.6 as well.

### CODE REVIEW

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 1]
```diff
@@ -3,9 +3,11 @@
...
 
-You can also test the procedure of provisioning multiple single-node OpenShift clusters on edge resources by using the zero touch provisioning feature, which is a Technology Preview feature that is available with {ocp}. For more information about that procedure, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation.
+There is no platform integration with the platform where you install the cluster, as the platform value is set to `platform=none`. A single-node OpenShift cluster contains only a single node, which hosts the control plane services and the user workloads. This configuration can be helpful when you want to minimize the resource footprint of the cluster. 
+
+You can also provision multiple single-node OpenShift clusters on edge resources by using the zero touch provisioning feature, which is a feature that is available with {ocp}. For more information about zero touch provisioning, see _Clusters at the network far edge_ in the {ocp-short} documentation.
```
Comment (berenss) [inline clusters/cluster_lifecycle/create_cluster_on_prem.adoc:11]: This will link to the latest topic, ie OCP 4.12?

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 2]
```diff
@@ -17,25 +19,28 @@ See the following prerequisites before creating a cluster in an on-premises envi
 
 * You must have a deployed hub cluster on {ocp-short} version 4.9, or later.
 * You need a configured infrastructure environment with a host inventory of configured hosts. 
-* You must have Internet access for your hub cluster (connected), or a connection to an internal or mirror registry that has a connection to the Internet (disconnected) to retrieve the required images for creating the cluster.
+* You must have internet access for your hub cluster (connected), or a connection to an internal or mirror registry that has a connection to the internet (disconnected) to retrieve the required images for creating the cluster.
 * You need a configured on-premises credential. 
-* You need an {ocp-short} image pull secret. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/images/managing-images#using-image-pull-secrets[Using image pull secrets] for more information.
-
+* You need an {ocp-short} image pull secret. See _Using image pull secrets_.
```
Comment (berenss) [inline clusters/cluster_lifecycle/create_cluster_on_prem.adoc:25]: same ask here, to make sure we land into the latest OCP 4.12 doc topic.

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 3]
```diff
@@ -49,13 +54,14 @@ Every managed cluster must be associated with a managed cluster set. If you do n
...
+The *OpenShift version* identifies the version of the {ocp-short} image that is used to create the cluster. If the version that you want to use is available, you can select the image from the list of images. If the image that you want to use is not a standard image, you can enter the URL to the image that you want to use. See _Release images_ to learn more.
 
-When you select an OpenShift version that is 4.9 or later, an option to select *Install single node OpenShift (SNO)* is displayed. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads. You cannot add additional nodes to a single-node OpenShift cluster after it is created. 
+When you select an OpenShift version that is 4.9 or later, an option to select *Install single-node OpenShift (SNO)* is displayed. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads. You cannot add additional nodes to a single-node OpenShift cluster after it is created.
```
Comment (berenss) [inline clusters/cluster_lifecycle/create_cluster_on_prem.adoc:60]: actually you can add nodes after a single node openshift is created. here is the doc topic for it: 
https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#scale-hosts-infrastructure-env

Comment (berenss) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc:60]: In fact the next doc steps seems to contradict "You cannot add additional nodes to a single-node OpenShift cluster after it is created." so I suggest we just remove that line.

**Review by berenss — APPROVED** [review]
great job with the changes!

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc:11]: Yep it links to 4.12 at the moment. I thought about 4.13 since this is for 2.8 but wasn't sure. Should we go with 4.13?

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc:60]: Fixed, let me know if it looks good now

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc (hunk 1 above)
Comment (berenss) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc:11]: in the 2.8 branch we should link to 4.13, but I know that is a moving target as the 4.13 docs are not yet public and will not be for another ~1 month.

Comment (oafischer) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc:11]: Sounds good, I'll go with 4.13 since that is the long term plan either way. Thanks!
